src/patternfly/components/JumpLinks/examples/JumpLinks.md

```diff
@@ -0,0 +1,306 @@
+        {{/jump-links-item-text}}
+      {{/jump-links-link}}
+    {{/jump-links-item}}
+    {{#> jump-links-item}}
+      {{#> jump-links-link}}
+        {{#> jump-links-item-text}}
+          Inactive section
+        {{/jump-links-item-text}}
+      {{/jump-links-link}}
+    {{/jump-links-item}}
+    {{#> jump-links-item}}
+      {{#> jump-links-link}}
+        {{#> jump-links-item-text}}
+          Inactive section
+        {{/jump-links-item-text}}
+      {{/jump-links-link}}
+    {{/jump-links-item}}
+    {{#> jump-links-item}}
+      {{#> jump-links-link}}
+        {{#> jump-links-item-text}}
+          Inactive section
+        {{/jump-links-item-text}}
+      {{/jump-links-link}}
+    {{/jump-links-item}}
+  {{/jump-links-list}}
+{{/jump-links}}
+```
+
+### Vertical with label
+```hbs
+{{#> jump-links jump-links--modifier="pf-m-vertical" jump-links--attribute='aria-label="Local"'}}
+  {{#> jump-links-label}}
+    Jump to section
+  {{/jump-links-label}}
+  {{#> jump-links-list}}
+    {{#> jump-links-item}}
+      {{#> jump-links-link}}
+        {{#> jump-links-item-text}}
+          Inactive section
+        {{/jump-links-item-text}}
+      {{/jump-links-link}}
+    {{/jump-links-item}}
+    {{#> jump-links-item jump-links-item--modifier="pf-m-current"}}
+      {{#> jump-links-link}}
+        {{#> jump-links-item-text}}
+          Active section
+        {{/jump-links-item-text}}
+      {{/jump-links-link}}
+    {{/jump-links-item}}
+    {{#> jump-links-item}}
+      {{#> jump-links-link}}
+        {{#> jump-links-item-text}}
+          Inactive section
+        {{/jump-links-item-text}}
+      {{/jump-links-link}}
+    {{/jump-links-item}}
+    {{#> jump-links-item}}
+      {{#> jump-links-link}}
+        {{#> jump-links-item-text}}
+          Inactive section
+        {{/jump-links-item-text}}
+      {{/jump-links-link}}
+    {{/jump-links-item}}
+    {{#> jump-links-item}}
+      {{#> jump-links-link}}
+        {{#> jump-links-item-text}}
+          Inactive section
+        {{/jump-links-item-text}}
+      {{/jump-links-link}}
+    {{/jump-links-item}}
+  {{/jump-links-list}}
+{{/jump-links}}
+```
+
+### Vertical with subsections
+```hbs
+{{#> jump-links jump-links--modifier="pf-m-vertical" jump-links--attribute='aria-label="Local"'}}
+  {{#> jump-links-label}}
+    Jump to section
+  {{/jump-links-label}}
+  {{#> jump-links-list}}
+    {{#> jump-links-item}}
+      {{#> jump-links-link}}
+        {{#> jump-links-item-text}}
+          Inactive section
+        {{/jump-links-item-text}}
+      {{/jump-links-link}}
+    {{/jump-links-item}}
+    {{#> jump-links-item jump-links-item--modifier="pf-m-current"}}
+      {{#> jump-links-link}}
+        {{#> jump-links-item-text}}
+          Active expandable section
+        {{/jump-links-item-text}}
+      {{/jump-links-link}}
+        {{#> jump-links-list newcontext}}
+          {{#> jump-links-item}}
+            {{#> jump-links-link jump-links-link--modifier="pf-m-subsection"}}
```

Comment:
I don't know about having to add a `pf-m-subsection` to every nested link. Seems like we could handle that with nested link selectors in the CSS, similar to tree view.